lib/matplotlib/widgets.py

```diff
@@ -1303,7 +1303,7 @@ def _release(self, event):
 
     def onmove(self, event):
         """Cursor move event handler and validator"""
-        if not self.ignore(event) and self.eventpress:
+        if not self.ignore(event):
```

Comment:
it is also a base class for other widgets, possibly subclassed indirectly by other developers. I haven't totally grokked the implications, but my knee-jerk reaction is that this could cause issues.